ui/lib/core/addon/components/form-field.hbs

```diff
@@ -6,7 +6,41 @@
 {{! template-lint-configure simple-unless "warn" }}
 <div class="field" data-test-field={{or @attr.name true}}>
   {{#if this.isHdsFormField}}
-    {{! TODO the template logic for the form fields migrated to HDS will appear here }}
+    {{#if @attr.options.possibleValues}}
+      {{#if (eq @attr.options.editType "checkboxList")}}
+        <Hds::Form::Checkbox::Group @name={{@attr.name}} data-test-input={{@attr.name}} as |G|>
+          {{#if this.labelString}}
+            <G.Legend data-test-form-field-label>{{this.labelString}}</G.Legend>
+          {{/if}}
+          {{#if this.showHelpText}}
+            <G.HelperText data-test-help-text>{{@attr.options.helpText}}</G.HelperText>
+          {{/if}}
+          {{#if @attr.options.subText}}
+            <G.HelperText data-test-label-subtext>
+              {{@attr.options.subText}}
+              {{#if @attr.options.docLink}}
+                <DocLink @path={{@attr.options.docLink}}>See our documentation</DocLink>
+                for help.
+              {{/if}}
+            </G.HelperText>
+          {{/if}}
+          {{#each @attr.options.possibleValues as |option|}}
+            <G.CheckboxField
+              checked={{includes option (get @model this.valuePath)}}
```

Comment:
@Mongey @hellobontempo in one of my tests/spikes I added a comment here that says: 

> TODO! here I've replaced `includes` with `eq` because it was not working: `includes` expects a list/array as second argument, while here we're passing a single value - see: https://github.com/DockYard/ember-composable-helpers?tab=readme-ov-file#includes

and then changed the line to
```
checked={{eq option (get @model this.valuePath)}}
```

But I imagine is working correctly in production, so probably nothing to see here. But I wanted to raise it anyway, because I noticed the comment in one of my branches and I wanted to double check with you (that have _a little bit_ more context/knowledge of the codebase :) )

Reply:
Hmm, we should check if this still works! When I implemented this originally `get @model this.valuePath` is an array. So it sounds like we might need to make the input more flexible 🤔

Reply:
Actually, since multiple items can be selected I think it makes sense to keep the model value type an array (and no need to make it flexible and accept different value types)